graylog2-web-interface/src/views/components/searchbar/StreamsFilter.jsx

```diff
@@ -5,9 +5,10 @@ import Select from 'components/common/Select';
 
 const StreamsFilter = ({ disabled, value, streams, onChange }) => {
   const selectedStreams = value.join(',');
+  const placeholder = 'Select streams the search should include. Searches in all streams if empty.';
   return (
-    <div style={{ position: 'relative', zIndex: 10 }} data-testid="streams-filter">
-      <Select placeholder="Select streams the search should include. Searches in all streams if empty."
+    <div style={{ position: 'relative', zIndex: 10 }} data-testid="streams-filter" title={placeholder}>
+      <Select placeholder={placeholder}
```

Comment:
I couldn't find a way to add the title prop to the select, so I just added it to the container. this will allow the user to read the title, even when the select will may not display the hole text due to `text-overflow: ellipsis`